_posts/2023-01-05-exploring-process-automation-opportunities.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Exploring Process Automation opportunities - RPA and low-code application development
+author: jjjsaha
+excerpt: I finished my masters' thesis on this subject last year. The research approached the topic from two different angles, RPA and low-code application development. The specific tools studied were Robot Framework and Microsoft Power Platform.
+  
+tags:
+  - thesis
+  - RPA
+  - low-code
+  - Robot Framework
+  - Power Platform
+---
+
+Last year I finished my masters’ thesis about process automation. The research approached the topic from two different angles, RPA, and low-code application development, more specifically [Robot Framework](https://robotframework.org/) and [Microsoft Power Platform](https://powerplatform.microsoft.com/en-us/). Contrary to the majority of current research material about the topic, this research didn’t focus on the experiences of business owners or IT-specialists, but instead attempted to clarify what kinds of process automation solutions have been created, what kinds of solutions are desired and what kinds of solutions are considered helpful, from the point of view of people working on the processes themselves. Here are the summarized findings of my research.
```

Comment:
what is RPA? you could expand the definition on first use

Reply:
Good point. Added the definition